theme/types.ts

```diff
@@ -0,0 +1,7 @@
+import type { CowProtocolTheme } from '@cowprotocol/types'
+
+declare module 'styled-components' {
```

Comment:
Sorry, I don't understand, why this declaration is here?

Reply:
Simplified it now, but still needed for module augmentation - otherwise TypeScript would error on theme prop access since it wouldn't know about our theme types.

Reply:
Just noticed that it's in the root, this was a mistake. Fixing now.

Reply:
OK addressed it now.